3.5/runtime/windowsservercore-1803/Dockerfile

```diff
@@ -13,7 +13,8 @@ RUN powershell -Command `
         Expand-Archive microsoft-windows-netfx3.zip; `
         Remove-Item -Force microsoft-windows-netfx3.zip; `
         Add-WindowsPackage -Online -PackagePath .\microsoft-windows-netfx3\microsoft-windows-netfx3-ondemand-package~31bf3856ad364e35~amd64~~.cab; `
-        Remove-Item -Force -Recurse microsoft-windows-netfx3
```

Comment:
I think we should defer rolling these cleanup changes within the runtiem Dockerfiles until the next time the runtime images are patched.  We don't want to rebuild the runtime image with this VS2019 SDK update.